src/dbnode/storage/index/block_bench_test.go

```diff
@@ -123,3 +124,10 @@ func (m mockOnIndexSeries) OnIndexSuccess(blockStart xtime.UnixNano) {
 func (m mockOnIndexSeries) OnIndexFinalize(blockStart xtime.UnixNano) {
 
 }
+func (m mockOnIndexSeries) OnIndexPrepare() {
+
+}
+
+func (m mockOnIndexSeries) NeedsIndexUpdate(indexBlockStartForWrite xtime.UnixNano) bool {
```

Comment:
can you just return false always? doesn't seem like you're updating the value ever

Reply:
Yeah fair, had it like this in case we wanted to be able to return different values in the future